documentation/asciidoc/accessories/camera/libcamera_apps_post_processing_tflite.adoc

```diff
@@ -144,3 +144,43 @@ The stage operates on a low resolution stream image of size 300x300. The followi
 `libcamera-hello --post-process-file object_detect_tf.json --lores-width 400 --lores-height 300`
 
 image::images/detection.jpg[Image showing detected objects]
+
+==== `segmentation_tf` stage
+
+`segmentation_tf` uses a Google MobileNet v1 model. The model file can be downloaded from https://tfhub.dev/tensorflow/lite-model/deeplabv3/1/metadata/2?lite-format=tflite[], whilst the labels file can be found in the `assets` folder, named `segmentation_labels.txt`.
+
+This stage runs on an image of size 257x257. Because YUV420 images must have even dimensions, the low resolution image should be at least 258 pixels in both width and height. The stage adds a vector of 257x257 values to the image metadata where each value indicates which of the categories (listed in the labels file) that the pixel belongs to. Optionally, a representation of the segmentation can be drawn into the bottom right corner of the image.
```

Comment:
Just out of curiosity, why does each TFLite model seem to operate on a different image size? :wink: